src/Faithlife.Math/Faithlife.Math.csproj

```diff
@@ -13,10 +13,10 @@
     <PackageProjectUrl>https://github.com/Faithlife/FaithlifeMath</PackageProjectUrl>
     <PackageReleaseNotes>https://github.com/Faithlife/FaithlifeMath/VersionHistory.md</PackageReleaseNotes>
     <RepositoryUrl>https://github.com/Faithlife/FaithlifeMath.git</RepositoryUrl>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/Faithlife/RepoName/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/Faithlife/RepoName</PackageProjectUrl>
-    <PackageReleaseNotes>https://github.com/Faithlife/RepoName/blob/master/VersionHistory.md#released</PackageReleaseNotes>
-    <RepositoryUrl>https://github.com/Faithlife/RepoName.git</RepositoryUrl>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/Faithlife/FaithlifeMath/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/Faithlife/FaithlifeMath</PackageProjectUrl>
+    <PackageReleaseNotes>https://github.com/Faithlife/FaithlifeMath/blob/master/VersionHistory.md#released</PackageReleaseNotes>
+    <RepositoryUrl>https://github.com/Faithlife/FaithlifeMath.git</RepositoryUrl>
```

Comment:
These four lines need to be merged with the four above, keeping the most recent version of each. This was a bad merge in 60741806af274aabc3791d04c800d9c36e6d60a3.

Reply:
Oh, I hadn't seen that. That really *is* strange that it's not a compiler error.